### PR TITLE
Add expiration cronjob to object service

### PR DESCRIPTION
**Pull request by helfi92**
Github Bug/Issue: Fixes https://github.com/taskcluster/taskcluster/issues/3712

### CODE REVIEW

File: services/object/procs.yml [hunk 1]
```diff
@@ -2,6 +2,11 @@ web:
   type: web
   command: node services/object/src/main.js server
   defaultReplicas: 0
+cleanup-expire-objects:
```
Comment (djmitche) [inline services/object/procs.yml:5]: Judging by the table in `monitoring.mdx`, we usually name this `expire`, so let's do the same here.

File: services/object/procs.yml [hunk 2]
```diff
@@ -2,6 +2,11 @@ web:
...
+  type: cron
+  schedule: '0 0 * * *'
+  deadline: 86400
+  command: node services/object/src/main cleanup-expire-objects
```
Comment (djmitche) [inline services/object/procs.yml:9]: Is there an easy way to make sure this doesn't run on production deployments, similar to the `defaultReplicas: 0` for the web proc?

**Review by djmitche — APPROVED** [review]
Looks good!

File: services/object/procs.yml (hunk 2 above)
Comment (helfi92) [reply services/object/procs.yml]: We can probably add an additional field `ignore` to ignore the cron command if `ignore` is set to true. Would you like me to add this?

**Pull request merged**